### ISSUE TITLE
Mission Complete Route Animation

### ISSUE THREAD

**Issue by jonfroehlich** [ProjectSidewalk/SidewalkWebpage#216]
I like the redesign of the mission complete route animation; however, three things:
- The just completed green route animation is _way_ too slow. It takes a very long time for a route to animate and by then, most users will have already clicked 'Continue.'
- Ideally, the green route animation would draw while the green mission complete part of the progress bar is animating (in other words, they would be in sync)
- The blue progress bar color is the same color as the Continue button, which is visually confusing.

![image](https://cloud.githubusercontent.com/assets/1621749/17856505/e28133d6-684b-11e6-9026-ed8f9c5a78f5.png)

**Comment by kotarohara** [comment]
Animation speed has been changed from 3 sec to 1 sec.

**Comment by kotarohara** [comment]
@Soben713 Please work on this.

**Comment by Soben713** [comment]
I changed a couple of things:

1. Replaced the blue with dark-grey both in the mission-progress-bar and the map.
2. Made the green a little darker.
3. Changed the map style to a bright and colorless one to make it easier to focus on the routes.
4. Changed the animation speed to 0.6s for each segment (it was previously 1s).
5. Made the path strokes thicker.

Images for comparison:

![image](https://cloud.githubusercontent.com/assets/1576529/20240498/af0db6f0-a8e7-11e6-8925-19d49750e249.png)
![image](https://cloud.githubusercontent.com/assets/1621749/17856505/e28133d6-684b-11e6-9026-ed8f9c5a78f5.png)

Thoughts?
